starfish/imagestack/imagestack.py

```diff
@@ -614,24 +617,24 @@ def _iter_tiles(
 
     def apply(
             self,
-            func,
-            split_by: Set[Indices]={Indices.X, Indices.Y},
+            func: Callable,
+            split_by: Set[Indices]=None,
             in_place=False,
             verbose: bool=False,
             n_processes: Optional[int]=None,
             **kwargs
     ) -> "ImageStack":
-        """Apply func over all tiles or volumes in self
+        """Apply func over all N-dimensional chunks of this image, yielding identically shaped
+        chunks, which are constituted into an ImageStack and returned.
```

Comment:
Oh man not sure about "chunks" what if it was just "Apply func over all N-dimensions of this image, and return the resulting imagestack"

Reply:
See #740 :)